src/pages/policies/policies.tsx

```diff
@@ -152,6 +175,10 @@ function PoliciesLayout(props: PoliciesLayoutProps) {
             block
           />
         </div>
+        <button type="submit" className={styles.searchButton}>
+          <SearchIcon aria-hidden />
+          <span className="sr-only">Submit</span>
```

Comment:
I am not sure what `sr-only` means.  Can you be more specific?

Reply:
I copied it over from Tailwind: https://tailwindcss.com/docs/screen-readers#screen-reader-only-elements

And it stands for "screen reader only", so it's for making this button accessible by giving it text for screen readers, as the icon is for visual users only 👍 (a positive side effect is that this button becomes easily testable as well)

This util also exists for Bootstrap